index.html

```diff
@@ -78,7 +106,7 @@ <h5>October, 2019</h5>
                                 <p>
                                     <small>21 October, 2019</small>
                                 </p>
-                                <a href="./batches/2019/career-camp-21-october-2019/index.html"
+                                <a href="./batches/2020/full-stack-web-dev-2020/index.html"
```

Comment:
Undo this change in line 109.

Reply:
hey pls tell me in brief what changes are there to be made?

Reply:
Sorry for the late response. Merging your branch. :)